docs/docs/cmd/spo/listitem/listitem-roleinheritance-reset.md

```diff
@@ -0,0 +1,39 @@
+# spo listitem roleinheritance reset
+
+Restores the role inheritance of list item, file, or folder
+
+## Usage
+
+```sh
+m365 spo listitem roleinheritance reset [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site from which the item role inheritance should be reset
```

Comment:
Let's simplify this to `URL of the site where the item for which to reset role inheritance is located`